lessons/lesson-3/project/how-to-style-bounding-box.md

```diff
@@ -1,2 +1,63 @@
+
+![Add Bounding Box Script component](../../../.gitbook/assets/how_to_style_bounding_box/bounding_box_script.PNG)
+
+Select the **Bounding Box** script to apply the component to the object. The bounding box is only visible in Game mode. Press play to view the bounding box.
+
+![View Bounding Box in Game Mode](../../../.gitbook/assets/how_to_style_bounding_box/bounding_box_default.PNG)
+
+## Change Wireframe Color
+
+You can change the color of the wireframe (the bounding box lines) by assigning a material to the **Wireframe Material** property.
+
+In the **Box Display** section, drag a new material to the **Wireframe Material** parameter.
+
+![Change Wireframe Material parameter](../../../.gitbook/assets/how_to_style_bounding_box/wireframe_material_pink.PNG)
+
+When you press play, the wireframe for the bounding box will be the color you dragged into the **Wireframe Material** parameter.
+
+![New Wireframe Material](../../../.gitbook/assets/how_to_style_bounding_box/wireframe_color.PNG)
+
+## Change Wireframe Thickness
+
+You can adjust the thickness of the wireframe by modifying the value for the **Wireframe Edge Radius** property.
+
+In the **Box Display** section, change the value for the **Wireframe Edge Radius** parameter to **0.01**.
+
+![Change Wireframe Edge Radius parameter](../../../.gitbook/assets/how_to_style_bounding_box/wireframe_edge_radius.PNG)
+
+When you press play, the wireframe for the bounding box will thicker than the default thinner wireframe.
+
+![New Wireframe thickness](../../../.gitbook/assets/how_to_style_bounding_box/wireframe_thickness.PNG)
+
+## Change Handle Color
+
+You can change the color of the handles by assigning a material to the **Handle Material** property.
+
+In the **Handles** section, drag a new material to the **Handle Material** parameter.
+
+![Change Handle Material parameter](../../../.gitbook/assets/how_to_style_bounding_box/handle_material.PNG)
+
+When you press play, the handles for the bounding box will be the color you dragged into the **Handle Material** parameter.
+
+![New Handle Material](../../../.gitbook/assets/how_to_style_bounding_box/handle_color.PNG)
+
+## Change Handle Color When Object is Grabbed
+
+You can change the color of the handles when an object is grabbed by assigning a material to the **Handle Grabbed Material** property.
+
+In the **Handles** section, drag a new material to the **Handle Grabbed Material** parameter.
+
+![Change Handle Grabbed material](../../../.gitbook/assets/how_to_style_bounding_box/handle_grabbed_parameter.PNG)
+
+When you press play, grab oen of the handles of the bounding box. The color of the handle will change to the color you dragged into the **Handle Grabbed Material** parameter.
+
+![New Handle Grabbed material](../../../.gitbook/assets/how_to_style_bounding_box/handle_grabbed_color.PNG)
```

Comment:
This section looks good but the end product(Cube with styled manipulation handlers) doesn't fit the mrtk style in the first picture. Can you make sure it does. Thanks!